spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/iam/TokenValidationService.java

```diff
@@ -1,10 +1,9 @@
-package org.eclipse.dataspaceconnector.token.spi;
+package org.eclipse.dataspaceconnector.spi.iam;
 
-import org.eclipse.dataspaceconnector.spi.iam.ClaimToken;
 import org.eclipse.dataspaceconnector.spi.result.Result;
 import org.jetbrains.annotations.NotNull;
 
+@FunctionalInterface
 public interface TokenValidationService {
-
     Result<ClaimToken> validate(@NotNull String token);
```

Comment:
Shouldn't the Token* classes be in a token package instead of IAM?

Reply:
There are already some existing *Token classes there (e.g. ClaimToken, TokenRepresentation) so for now I guess it's ok to put them all together in the IAM package.